.travis.yml

```diff
@@ -53,3 +53,9 @@ script:
   - grunt lint
   - travis_retry npm run test-travis
   - travis_retry npm run test-server
+
+matrix:
+  allow_failures:
+    - node_js: "0.12"
+    - node_js: "iojs"
+  fast_finish: true
```

Comment:
My understanding is that fast_finish aborts if there are *any* failures, yet [travis_retry](https://github.com/mozilla/fxa-content-server/pull/2371/files#diff-354f30a63fb0907d4ad57269548329e3R54) retries 3 times. Am I confused here?